src/ima-state-machine.js

```diff
@@ -295,6 +286,15 @@ function onAdBreakStart(options: Object, adEvent: any): void {
   this.player.pause();
   const adBreakOptions = getAdBreakOptions.call(this, adEvent);
   const adBreak = new AdBreak(adBreakOptions);
+  // When we are using the same video element on iOS, native captions still
+  // appearing on the video element, so need to hide them before ad start.
+  if (this.playOnMainVideoTag()) {
```

Comment:
this actually makes sense cause they say they load the ad 4 sec before in IMA SDK docs.
Also note that for postroll they load it even way before, right after last midroll finishes, and if there's no midroll and only preroll, so after preroll.
The issue is just the hiding though, but we can keep all logic in same place.
Maybe put this in separate `_maybeHideTextTrack` method and call it from here to keep the convention.

Reply:
Did they write something about when they have changed the timing of the loading ads?

Reply:
I don't remember but it should appear in the release notes they publish

Reply:
Looked for it in their release notes and I didn't find it.. super weird

Reply:
I remember seeing this lately but couldn't find it as well.